mkdocs.yml

```
@@ -29,8 +29,8 @@ markdown_extensions:
         permalink: true
     - attr_list
     - pymdownx.emoji:
-        emoji_index: !!python/name:materialx.emoji.twemoji
-        emoji_generator: !!python/name:materialx.emoji.to_svg
+        emoji_generator: ""
+        emoji_index: ""
```

Not sure why this change is getting into the generated code 🤔 
```suggestion
        emoji_index: !!python/name:materialx.emoji.twemoji
        emoji_generator: !!python/name:materialx.emoji.to_svg
```

---

Yes, even I found it weird then